### PR TITLE
NEXT-8427 - EAN rename to GTIN

### PR DESCRIPTION
**Pull request by JensBrunken**
### 3. Checklist

- [ ] I have written tests and verified that they fail without my change
- [ ] I have squashed any insignificant commits
- [x] I have created a [changelog file](https://github.com/shopware/platform/blob/master/adr/2020-08-03-Implement-New-Changelog.md) with all necessary information about my changes
- [x] I have written or adjusted the documentation according to my changes
- [ ] This change has comments for package types, values, functions, and non-obvious lines of code
- [ ] I have read the contribution requirements and fulfil them.

### CODE REVIEW

File: CHANGELOG-6.3.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ NEXT
 ----------------
 
 #### Administration
+* Added `GTIN` label to the detail page
```
Comment (mitelg) [inline CHANGELOG-6.3.md:23]: please read the contribution guideline regarding the changelog!

Comment (shyim) [inline CHANGELOG-6.3.md:23]: Please see https://github.com/shopwareBoostDay/platform/tree/master/changelog

File: src/Administration/Resources/app/administration/src/module/sw-product/snippet/de-DE.json [hunk 2]
```diff
@@ -98,7 +98,7 @@
       "placeholderMaxPurchase": "Maximalabnahme ...",
       "placeholderPurchaseSteps": "Verkaufsintervall (Stückzahl) ...",
       "placeholderRestockTime": "Wiederauffüllintervall ...",
-      "placeholderEan": "Gib die EAN-Nummer ein ...",
+      "placeholderEan": "Gib die GTIN ein ...",
```
Comment (mitelg) [inline src/Administration/Resources/app/administration/src/module/sw-product/snippet/de-DE.json:101]: we should also change the snippet keys

**Comment by JensBrunken** [comment]
new commit with the changes

**Comment by mitelg** [comment]
we are currently discussing if renaming the snippet keys is a break :see_no_evil: 

we will let you know

**Comment by tinect** [comment]
Is there any reason this PR doesn't fix all occurrences?

**Comment by tinect** [comment]
> we are currently discussing if renaming the snippet keys is a break 🙈
> 
> we will let you know

Idea: adding NEW key with gtin in name and mark old one deprecated in changelog?

**Comment by shyim** [comment]
As not all snippets of ean are changed, I will close this PR. Feel free to create an additional new one at shopware/platform

**Pull request closed without merging**